src/modules/game.js

```diff
@@ -0,0 +1,35 @@
+class Game {
+  constructor() {
+    this.id = 'hD3SsjUFdslkX5LTjKxj';
+    this.gameName = 'wakanda 2023 game';
+    this.url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games';
+  }
+
+  async getScore() {
+    try {
+      const response = await fetch(`${this.url}/${this.id}/scores`);
+      const scores = await response.json();
+      return scores.result;
+    } catch (error) {
+      return [];
```

Comment:
**The functionality of this app is perfect!**

- [ ] However in readme file, the MIT license like looks broken. Kindly fix it and make sure you are directing user to your correct MIT license link. Thank you!